spec.md

```diff
@@ -531,23 +531,51 @@ Credentials are a set of information used by an application or a user to utilize
 
 #### Log Drain
 
-There are a class of service offerings that provide aggregation, indexing, and analysis of log data. To utilize these services an application that generates logs needs information for the location to which it will stream logs. If a broker represents one of these services, it MAY return a `syslog_drain_url` in the response for a request to create a service binding, to which logs MUST be streamed.
+There are a class of service offerings that provide aggregation, indexing, and
+analysis of log data. To utilize these services an application that generates
+logs needs information for the location to which it will stream logs. A
+create binding response from a service broker that provides one of these
+services MUST include a `syslog_drain_url`. The platform MUST use the
+`syslog_drain_url` value when sending logs to the service.
 
-The `requires` field in the [Catalog](#catalog-management) endpoint enables a platform marketplace to validate a response for create binding that includes a `syslog_drain_url`. Platform marketplaces can consider a broker's response invalid if it includes a `syslog_drain_url` and `"requires":["syslog_drain"]` is not present in the [Catalog](#catalog-management) endpoint.
+Brokers MUST NOT include a `syslog_drain_url` in a create binding response
+if it did not include a `"requires":["syslog_drain"]` property in its
+[Catalog](#catalog-management) endpoint.
 
 #### Route Services
 
-There are a class of service offerings that intermediate requests to applications, performing functions such as rate limiting or authorization. To configure a service instance with behavior specific to an application's routable address, a broker client MAY send the address along with the request to create a binding using `"bind_resource":{"route":"some-address.com"}`.
+There are a class of service offerings that intermediate requests to
+applications, performing functions such as rate limiting or authorization.
+If the platform and service both support this feature then the following
+applies:
 
-Some platforms MAY support proxying of application requests to service instances. In this case the platform needs to know where to send application requests; to facilitate this, the broker MUST return a `route_service_url` in the response for a request to create a binding. Not all services of this type expect to receive requests proxied by the platform; some services will have been configured out-of-band to intermediate requests to applications. In this case, the broker will not return `route_service_url` in response to the create binding request. By sending `bind_resource` as described above, the platform enables dynamic configuration of a service instance already in the application request path for the route, requiring no change in the platform routing tier.
+- To configure a service instance with behavior specific to an application's
```

Comment:
This section reads a bit like sending the `bind-resource` field is optional based on whether the platform supports configuration of the instance specific to the routable address.

As this is up to the broker author, we should simply say that the platform must send `bind-resource` if it supports Route Services. The broker author may choose to support configuration by routable address, or not. A broker could implement service instances that perform the same operation no matter what the address, but that require the route to perform the operation at all.

Suggestion:
"If a platform supports Route Services, it MUST send a routable address or endpoint for the application along with the request to create a service binding using `"bind_resource":{"route":"some-address.com"}`. In the case that a platform supports binding multiple routable addresses to the same service instance, the broker author MAY support configuration specific to an address using `parameters`."